keyboards/claw44/keymaps/yfuku/rules.mk

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Same as the `rules.mk` file at `keyboards/claw44/keymaps/default/`, this whole file's contents should be in `keyboards/claw44/rev1/rules.mk` instead, and this file either zero-length or deleted entirely.